### PR TITLE
Creating French Swahili Congo Baseline

### PR DESCRIPTION
**Pull request by espoirMur**
# French to Swahili Congo

Author: Murhabazi B. Espoir

## Background

Swahili Congo is quite different from the Swahili spoken in Kenyan.
It's spoken in the large part Eastern Congo, in Burundi and Rwanda.
Since many people have french as their official language I found that it may be useful to French to Swahili Congo

## Data

	- The JW300 French-Swahili Congo.

## Model

	- Default Masakhane Transformer translation model.
	- The model can be found [here](http://34.66.168.159/french_kiswhahili_espoir/fr-swc-baseline_v2/models/frswc_transformer/best.ckpt)

## Analysis

Example 1
```
    Source:     Avant de passer en cour martiale , j’ai été transféré dans un camp militaire à Héraclion , en Crète .
	Reference:  Kabla ya kupelekwa kortini , nilihamishiwa kwenye kambi ya kijeshi huko Iráklion , Krete .
    Hypothesis: Kabla ya kuingia katika makao ya kijeshi , nilihamishwa hadi kambi ya kijeshi huko Heraclion , Krete .
```

Example 2
```
	Source:     ● La médecine connaît un revirement funeste : certains médicaments “ miracles ” pourraient avoir perdu leur pouvoir .
	Reference:  Magonjwa ya kuambukiza yaliyokuwa yakiwaua mamilioni ya watu miaka mingi iliyopita kama vile , ukoma na kifua kikuu , yalidhibitiwa kwa dawa za kuua viini .
	Hypothesis: ● Mabadiliko makubwa ya kitiba yanatokea , yaani , huenda dawa fulani “ za kimuujiza ” zikapoteza uwezo wao .
```

Example 3
```
Source:     Lisons - ​ nous la Parole de Dieu chaque jour ?
Reference:  Je , tunapendezwa na sheria ya Mungu na kusoma Neno lake kila siku ?
Hypothesis: Je , tunasoma Neno la Mungu kila siku ?
```
## Results
	- BLEU dev :   26.73
	- BLEU test :  33.73

### CODE REVIEW

File: fr-swc/jw300-baseline/baseline.ipynb [hunk 1]
```diff
@@ -0,0 +1,4443 @@
+{
```
Comment (jaderabbit) [inline fr-swc/jw300-baseline/baseline.ipynb]: <p>We should save these 643 in the global test set</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (juliakreutzer) [reply fr-swc/jw300-baseline/baseline.ipynb]: Agreed!

**Comment by espoirMur** [comment]
> Please add the 643 sentences to the global test file and add `test.fr-swc.fr` and `test.fr-swc.swc` to the jw300 test data directory (https://github.com/masakhane-io/masakhane/tree/master/jw300_utils/).
> 
> Thank you! Great to see benchmarks with French coming along! 😎

I have fixed this @juliakreutzer 🤦🏾‍♂️

File: fr-swc/jw300-baseline/baseline.ipynb (hunk 1 above)
Comment (espoirMur) [reply fr-swc/jw300-baseline/baseline.ipynb:1]: This is done @jaderabbit !!🤦🏾‍♂️

**Pull request merged**